src/components/tiptap/autofieldSelector/autofillMenuSuggestion.ts

```diff
@@ -17,7 +17,7 @@ let customFields: ICustomField[] = []
 
 const token = new URLSearchParams(document?.location?.search).get('token')
 ;(async () => {
-  customFields = await getCustomFields(token || '')
+  customFields = await getCustomFields(token as string)
```

Comment:
A zod validation would be nice. 

`z.string().parse(token)`

Reply:
next commit resolves this